src/main/drivers/dshot_bitbang_decode.c

```diff
@@ -115,7 +115,10 @@ uint32_t decode_bb_bitband( uint16_t buffer[], uint32_t count, uint32_t bit)
         // not returning telemetry is ok if the esc cpu is
         // overburdened.  in that case no edge will be found and
         // BB_NOEDGE indicates the condition to caller
-        return BB_NOEDGE;
+		
+		// test
+        //return BB_NOEDGE;
+		return BB_INVALID;
```

Comment:
This seems to be test code only?

Reply:
Yes, this is for test. I think we can count all kinds of errors this way. I though to open this disscussion. I am not sure it is proper for a PR. Maybe I should just open an issue first?

Reply:
I see. By default pull requests are used to propose changes to the code that the author thinks should actually be implemented. So if you want to open a pull request for something that is for testing / discussion only then you should indicate this in the title or text of the pull request.

Reply:
Thank you so much. I will make a proper title for a PR next time.
I have some test results:
16.71   error: 13.36%
16.72   error:   0.73%
16.72a error:   0.36%  (timming fine tuned with this new error counting function)
Jesc      error: 11.95%
BL32     error:   0.0% (another test setting from Asizon)